README.md

```diff
@@ -63,27 +63,29 @@ Every voice is important and every idea is valuable. If you have something on yo
 
 ### ansible-core
 
-- 2.11
 - 2.12
 - 2.13
+- 2.14
 - current development version
 
 ### Databases
 
 - mysql 5.7.40
 - mysql 8.0.31
-- mariadb:10.4.24 (only collection version >= 3)
-- mariadb:10.5.18 (only collection version >= 3)
-- mariadb:10.6.11 (only collection version >= 3)
-- mariadb:10.7.7 (only collection version >= 3)
-- mariadb 10.8.6 (only collection version >= 3)
-- mariadb 10.9.4 (only collection version >= 3)
+- mariadb:10.4.24 (only collection version >= 3.5.1)
+- mariadb:10.5.18 (only collection version >= 3.5.1)
+- mariadb:10.6.11 (only collection version >= 3.5.1)
+- mariadb:10.7.7 (only collection version >= 3.5.1)
+- mariadb 10.8.6 (only collection version >= 3.0)
 
```

Comment:
@laurent-indermuehle great work, thanks you

Do you think we need to test against 5 versions of mariadb?
How about testing against the oldest and the latest supported ones?
It would dramatically reduce the number of jobs and total CI time respectively.

Reply:
First, sorry to disappoint you as I was last week: Removing tests don't spare time because GHA runs them in parallel :(

I forgot to check the [release policy](https://mariadb.org/about/#:~:text=MCA%20or%20BSD.-,Maintenance%20policy,until%20at%20least%20November%202023.) before posting my PR:

- we can remove 10.7 and 10.8 as they are note long-term releases
- I removed 10.3 because it will be EoL in 4 months

So if we stick to long-term release we can test :
- 10.4
- 10.5
- 10.6
- 10.11 when released

I'll commit that change soon.

Reply:
btw I'm not disappointed as ci speed is not the most important thing:)